src/workerd/jsg/promise.h

```diff
@@ -383,19 +386,17 @@ class Promise {
       resolve(Lock::from(deprecatedIsolate), kj::mv(value));
     }
     template <typename U = T, typename = kj::EnableIf<isVoid<U>()>>
-    void resolve() {
+    void resolve() KJ_DEPRECATED("Use the jsg::Lock& taking variant instead") {
```

Comment:
is the plan to remove these completely?

Reply:
Yes, once I'm sure everything that is using them currently have been updated.